### ISSUE TITLE
[Bug] SwipeView removes Frame borders on Android

### ISSUE THREAD

**Issue by EmilAlipiev** [xamarin/Xamarin.Forms#11831]
### Description
When using Frame inside SwipeView, Frame borders are not working. Frame appears without borders.
### Steps to Reproduce

Use the code and template as below,

```
 
   <Style x:Key="CollectionSwipeSelectedStyle" TargetType="SwipeView">
                <Setter Property="VisualStateManager.VisualStateGroups">
                    <VisualStateGroupList>
                        <VisualStateGroup x:Name="CommonStatesSwipe">
                            <VisualState x:Name="Normal">
                                <VisualState.Setters>
                                    <Setter Property="BackgroundColor" Value="White" />
                                </VisualState.Setters>
                            </VisualState>
                            <VisualState x:Name="Selected">
                                <VisualState.Setters>
                                    <Setter Property="BackgroundColor" Value="LightGray" />
                                </VisualState.Setters>
                            </VisualState>
                        </VisualStateGroup>
                    </VisualStateGroupList>
                </Setter>
            </Style>

      <CollectionView.ItemTemplate>
                <DataTemplate>
                    <SwipeView Style="{StaticResource CollectionSwipeSelectedStyle}">
                        <SwipeView.RightItems>
                            <SwipeItems>
                                <SwipeItemView
                                    BackgroundColor="#DB0000"
                                    Command="{Binding Path=BindingContext.DeleteCmd, Source={x:Reference Name=CollectionView}}"
                                    CommandParameter="{Binding .}"
                                    HeightRequest="83"
                                    WidthRequest="77">                        
                                </SwipeItemView>
                            </SwipeItems>
                        </SwipeView.RightItems>
                        <Frame
                            Padding="1"
                            BorderColor="Silver"
                            WidthRequest="588"> 
 
```

### Expected Behavior
Borders to appear
### Actual Behavior
without borders
### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
 
  - Android:  9
 
- Nuget Packages: XF 4.8
- Affected Devices: Android 9 emulator

**Comment by jsuarezruiz** [comment]
@EmilAlipiev  Do you mean the border color?
![issue11831](https://user-images.githubusercontent.com/6755973/90524605-8e672d00-e16e-11ea-81dd-9fd5565161a7.gif)

**Comment by jsuarezruiz** [comment]
@EmilAlipiev Thanks for the feedback. After apply your style with VSM, etc. I understand what happens. Will be fixed by #11947
